docs/user/exercises/programming-submission-policy-configuration.inc

```diff
@@ -0,0 +1,89 @@
+.. _configure_submission_policies:
+
+The submission policy defines the effect that a submission has on the participation of one participant in a programming exercise.
+A programming exercise might have no submission policy at all, or one submission policy, but never more than one. Submission policies are initially specified
+in the creation process of a programming exercise and can later be adjusted in the grading configuration of the particular programming exercise.
+
+.. note::
+    One submission is defined by one push to the exercise participation repository by the participant themself that triggers the
+    automatic tests resulting in feedback for the participant. Automatic test runs triggered by instructors are not considered submissions.
+
+Submission Policy Configuration & General Information
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Choosing the right submission policy configuration depends on the exercise and your teaching style.
+In general, lock repository and submission penalty policies combat trial-and-error solving approaches.
+
+**1. None**
+When selecting no submission policy, exercise participants can submit their solution as often as they want until the deadline.
+
+.. figure:: programming/submission-policy-none-form.png
+          :align: center
+
+**2. Lock Repository**
+Participants can submit a fixed number of times within the submission period of a programming exercise. Once the participant reaches the submission limit,
+further participation in the exercise is prevented by locking the participation repository. The participant may still work on their solution. However, Artemis
+does not provide further feedback and does not change the participation score.
```

Comment:
> The participant may still work on their solution. However, Artemis does not provide further feedback and does not change the participation score.

How does that work? Just working locally? If locked, I cannot submit again for a final grading e.g. after the deadline, right?
I'd adjust this or add a statement that clearly states where additional submissions are possible at all. (Probably not?)

Reply:
You're right, the formulation is not really clear here. I reformulated this to make it more clear that submissions are not possible after the submission is locked. 

`Participants can submit a fixed number of times within the submission period of a programming exercise. Once the participant reaches the submission limit, further participation in the exercise is prevented by locking the participation repository. The participant may still work on their solution locally, but cannot submit it to Artemis to receive feedback.`

Do you think it is more clear now? @jpbernius